scripts/rollup-replace.js

```diff
@@ -39,6 +39,11 @@ export default function(options) {
 
             code = globalReplace(code, 'rotateLeft', 'scope.rotateLeft');
             code = globalReplace(code, 'rotateUp', 'scope.rotateUp');
+            // comment out the context menu prevent default line...
+            code = globalReplace(code,
+              "scope.domElement.addEventListener\\( 'contextmenu'",
```

Comment:
This comments out https://github.com/mrdoob/three.js/blob/dev/examples/js/controls/OrbitControls.js#L226 so that we don't preventDefault on the contextmenu. It makes sense to do this if you are using pan controls with right click, but we are not.